featurebyte/api/context.py

```diff
@@ -0,0 +1,80 @@
+"""
+UseCase module
+"""
+from __future__ import annotations
+
+from typing import Any, List
+
+from featurebyte.api.savable_api_object import SavableApiObject
+from featurebyte.common.doc_util import FBAutoDoc
+from featurebyte.models.base import PydanticObjectId
+from featurebyte.models.context import ContextModel
+from featurebyte.schema.context import ContextCreate, ContextUpdate
+
+
+class Context(SavableApiObject):
+    """
+    Context class to represent a Context in FeatureByte.
+
+    A context defines the circumstances in which features are expected to be served.
+    """
+
+    __fbautodoc__ = FBAutoDoc(proxy_class="featurebyte.Context")
+
+    # class variables
+    _route = "/context"
+    _list_schema = ContextModel
+    _get_schema = ContextModel
+    _update_schema_class = ContextUpdate
+    _list_fields = [
+        "name",
+        "entity_ids",
+        "description",
+    ]
+
+    # pydantic instance variable (public)
+    entity_ids: List[PydanticObjectId]
```

Comment:
If we do the following:
```
context = Context(...).save()

another_context = catalog.get_context_by_id(context.id)
print(another_context.entity_ids)
```
I would expect the output will be empty `entity_ids`. We add add a property `entity_ids` that retrieve it from the `cached_model`.

Reply:
Seems like we have the test to cover this. As long as `entity_ids` is immutable after creation, this should be fine.